internal/sql/repository/AppListingRepository.go

```diff
@@ -510,3 +511,18 @@ func (impl AppListingRepositoryImpl) DeploymentDetailByArtifactId(ciArtifactId i
 
 	return deploymentDetail, nil
 }
+
+func (impl AppListingRepositoryImpl) FindAppCount(isProd bool) (int, error) {
+	var count int
+	query := "SELECT count(distinct pipeline.app_id) from pipeline pipeline " +
+		" INNER JOIN environment env on env.id=pipeline.environment_id" +
+		" INNER JOIN app app on app.id=pipeline.app_id" +
+		" WHERE env.default = ? and app.active=true;"
+	_, err := impl.dbConnection.Query(&count, query, isProd)
+	if err != nil {
+		impl.Logger.Errorw("Exception caught:", err)
```

Comment:
proper error logging

Reply:
added this "exception caught inside repository for fetching app count"

Reply:
done

Reply:
need to print stack trace